.github/workflows/build.yml

```diff
@@ -31,8 +31,10 @@ jobs:
           disable-sudo: true
           egress-policy: block
           allowed-endpoints: >
+            api.adoptium.net:443
             github.com:443
             jitpack.io:443
+            objects.githubusercontent.com:433
```

Comment:
I noticed the port number should be `443`, but is currently being set to `433` which is causing the outbound call to get blocked.